### DISCUSSION TITLE
میزان استفاده از منابع سیستم

### DISCUSSION THREAD

**Discussion by akv79** [hiddify/Hiddify-Manager#199] [Q&A سوالات رایج] [answered]
سلام خسته نباشید.
میخواستم بدونم یک هسته سی پی یو حدودا تا چند تا یوزر رو جواب میده؟
به عنوان مثال یک هسته سی پی یو و یک گیگ رم 50 تا یوزر رو جواب میده؟
با تشکر.

**Comment by arman2012** [comment, marked as answer]
بستگی به استفاده داره ولی بله تا صد تا هم میشه
